test/unit/ocsp/test_unit_ocsp_mode.js

```diff
@@ -9,13 +9,13 @@ const assert = require('assert');
 describe('OCSP mode', function () {
   it('getOcspMode', function (done) {
     // insecure mode
-    GlobalConfig.setInsecureConnect(true);
+    GlobalConfig.setDisableOCSPChecks(true);
```

Comment:
Can we have also a test confirming that `insecureMode` still works?

Reply:
I made testing for insecureConnect. Thank you.